iphone/Classes/TiUITableView.m

```diff
@@ -1639,7 +1639,8 @@ -(void)updateSearchView
 
 	if (tableHeaderView == nil)
 	{
-		CGRect wrapperFrame = CGRectMake(0, 0, [tableview bounds].size.width, TI_NAVBAR_HEIGHT);
+    CGFloat wrapperHeight = [TiUtils isIOS11OrGreater] ? 56.0 : 44.0;
```

Comment:
No hardcoded Values please.

Reply:
You mean I should use macro for same similar to previous implementation ?

Reply:
Yep. Or if possible get the height from the actual parent frame. Hardcoded values are the worst, we should try to avoid it at any time.